src/utils/constants.ts

```diff
@@ -8,15 +8,15 @@ export const MAX_NOTIFICATIONS = 3;
 export const ORCID_OAUTH_URL = `${process.env.REACT_APP_ORCID_BASE_URL}/oauth/token/`;
 export const ORCID_SIGN_IN_URL = `${process.env.REACT_APP_ORCID_BASE_URL}/signin?oauth&client_id=${process.env.REACT_APP_ORCID_CLIENT_ID}&response_type=code&scope=/authenticate&redirect_uri=${process.env.REACT_APP_ORCID_REDIRECT_URI}`;
 
-export enum ExternalApi {
-  PUBLICATION_CHANNEL = 'https://api.nsd.no/dbhapitjener/Tabeller/hentJSONTabellData',
-}
+const { REACT_APP_AUTHORITY_REGISTER_API_URL, REACT_APP_PUBLICATION_CHANNEL_API_URL } = process.env;
+
```

Comment:
hvorfor står det dobbelt opp?

Reply:
er det ikke bedre å skrive process.env.REACT_APP_BLABLA ?

Reply:
tanken var å bruke destructuring for å unngå å bruke `process.env` to ganger, men enig i at det egentlig bare ble mer rot.